core/dive.c

```diff
@@ -1390,10 +1401,12 @@ static void fixup_airtemp(struct dive *dive)
 
 /* zero out the airtemp in the dive structure if it was just created by
  * running fixup on the dive. keep it if it had been edited by hand */
```

Comment:
that comment is now misleading... 
```
/* if the air temperature in the dive data is redundant to the one in its
 * first divecomputer (i.e., it was added by running fixup on the dive)
 * return 0, otherwise return the air temperature given in the dive */
```

Reply:
Copied that verbatim, thanks.